docs/DeveloperGuide.md

```diff
@@ -1,38 +1,135 @@
-# Developer Guide
+# Developer Guide for SpendSwift
 
 ## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
-## Design & implementation
-
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
-
-## Product scope
-### Target user profile
-
-{Describe the target user profile}
-
-### Value proposition
-
-{Describe the value proposition: what problem does it solve?}
+{List here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well.}
+
+## Design & Implementation
+### Sequence Diagrams
```

Comment:
The following applies to **ALL** sequence diagrams:

According to the lecture notes, the UML notation for the sequence diagrams doesn't include the entity/object at the bottom of the lifeline. 

To rectify this, maybe you could include `hide footbox` at the top of your .puml files of your sequence diagrams?? This should resolve this issue. :)